File: framework/languages/java/e2e/SpotService/Client/src/main/java/systems/zlink/e2e/spotservice/client/Scenarios/SmE3Scenario.java
package systems.zlink.e2e.spotservice.client.Scenarios;


public final class SmE3Scenario extends SpotServiceScenarioContext {
    private SmE3Scenario(SpotServiceScenarioContext context) {
        super(context);
    }

    public static void run(SpotServiceScenarioContext context) {
        new SmE3Scenario(context).execute();
    }

    private void execute() {
        sleep(1200);
        System.out.println("scenario SM-E3 passed");

    }
}

File: framework/languages/java/e2e-kotlin/SpotService/Client/src/main/kotlin/systems/zlink/e2e/kotlin/spotservice/client/scenarios/SmE3Scenario.kt
package systems.zlink.e2e.kotlin.spotservice.client.scenarios

import systems.zlink.e2e.kotlin.spotservice.client.support.sleep

internal object SmE3Scenario {
    suspend fun run() {
        sleep(1200)
        println("scenario SM-E3 passed")
    }
}
